libs/gitlabl/sanitize_title.py

```diff
@@ -0,0 +1,27 @@
+'''
+This script contains function to sanitize the title for gitlab branches.
+'''
+import re
+
+from libs.kafka.logging import LogMessage
+
+
+def sanitize_title(unsanitized_title, servicename):
+    '''
+    sanitize_title will take an unsanitized title as input
+    and removes any special chars, which could possible violate
+    the gitlab branch naming rules.
+    @param unsanitized_title will be the unsanitized title in string format.
+        @param servicename will be the name of the service calling this function.
+    @return a sanitized title without any special chars except - in string format.
+    '''
+    sanitized_title = ''
```

Comment:
Return another default value. This will cause a problem when multiple empty strings will be returned.

Reply:
Used the unsanitized title as default value. So we won't have duplicates at least.

Reply:
The default value got md5 hashed.